gameplan/hooks.py

```diff
@@ -229,3 +229,24 @@
 # Recommended only for DocTypes which have limited documents with untranslated names
 # For example: Role, Gender, etc.
 # translated_search_doctypes = []
+
+frappe_search_doctypes = {
```

Comment:
are there any other configuration options you can think of that might come in the future? then we can rename this hook name like this:

```py
frappe_search = {
  'doctypes': {
    'title_field': 'title',
    'content_field': 'content',
    'payload': ['team', 'project']
  }
}
```

Reply:
We already have the index path, incidentally. Hard-coded right now, but must change. Probably a hook?

Isn't this a little too nested? If there are only two variables, we can just have `frappe_search_doctypes` and `frappe_search_index`. If we have more configuration later on (though I see no obvious cases), `frappe_search_config` instead of the index variable.

Reply:
Resolved in the repo PR